lib/columns_matching.rb

```diff
@@ -12,12 +12,16 @@ def initialize(options = {})
   #
   # @return [Array] of supported keys
   def supported_keys
-    @supported_keys ||= @content.fetch('mapping')
+    @supported_keys ||= @content.keys
+  end
+
+  def column_type(key)
+    @content[key].to_sym
   end
 
   private
 
   def parse_content_file
-    YAML.load_file('config/columns_match.yml')
+    YAML.load_file('config/columns_match.yml').fetch('mapping')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.